draft-ietf-httpbis-priority.md

```diff
@@ -634,6 +634,34 @@ signal the intended priority to the client by including the Priority field in
 a PUSH_PROMISE or HEADERS frame.
 
 
+# Retransmission Scheduling
+
+Transport protocols such as TCP and QUIC provide reliability by detecting packet
```

Comment:
Why is this not referring to the QUIC RFCs?

Reply:
Hah, thanks @reschke for promoting my H-index.

```suggestion
Transport protocols such as TCP {{?RFC0793}} and QUIC {{?RFC9000}} provide reliability by detecting packet
```